app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -216,6 +216,13 @@ a.inplace-editing--trigger-link,
     .inplace-edit--icon-wrapper
       visibility: visible
 
+.attributes-group
+  .inplace-edit
+    .inplace-edit--read-value
+      .ng-binding
+        p
+          font-size: inherit
+
```

Comment:
Is this deep a nesting actually necessary?
I believe it would also be possible to add a rule to the `.inplace-edit--read-value` block at https://github.com/opf/openproject/blob/dev/app/assets/stylesheets/content/_in_place_editing.sass#L165.

The `ng-binding` class is not a good selector btw, as it is generated by angular for angular empowered dom elements and it does not carry any semantic within the application.

A second rule could limit the margin for the last p element so that such custom fields, and the description as well, do not have weird empty space if there is content in them.